app/design/frontend/Magento/luma/Magento_Checkout/web/css/source/module/checkout/_payments.less

```diff
@@ -118,6 +118,7 @@
             .primary {
                 .action-update {
                     margin-right: 0;
+                    margin-bottom:20px;
```

Comment:
Hi @GovindaSharma. Thanks for collaboration. Please add indent between value and property declaration.